tests/ios/xcodeproj/pre_build_check.sh

```diff
@@ -2,26 +2,26 @@ set -eu
 
 cd $(dirname $0)
 
-export HSP=`grep "HEADER_SEARCH_PATHS" *.xcodeproj/project.pbxproj | grep -o  "bazel-out\S*\.hmap"`
+export HSP=$(grep "HEADER_SEARCH_PATHS" *.xcodeproj/project.pbxproj | grep -o "bazel-out\S*\.hmap")
 echo "Ensure hmap files do not exist (remove if they do)"
 for path in ${HSP}; do
     FULL_PATH="../../../$path"
     if [ -f $FULL_PATH ]; then
-        rm $FULL_PATH;
-        echo "Removed file at $FULL_PATH";
+        rm -f $FULL_PATH
+        echo "Removed file at $FULL_PATH"
     fi
 done
 
-export FSP=`grep "FRAMEWORK_SEARCH_PATHS" *.xcodeproj/project.pbxproj | grep -o "bazel-out[^ \\]*"`
+export FSP=$(grep "FRAMEWORK_SEARCH_PATHS" *.xcodeproj/project.pbxproj | grep -o "bazel-out[^ \]*")
```

Comment:
Did you remove the double-slash intentionally? Does wrapping this command in `$(` and `)` cause the grep expression to be evaluated differently?

Reply:
removed by `shfmt`